superset-frontend/src/explore/components/ExploreActionButtons.tsx

```diff
@@ -42,18 +44,27 @@ type ExploreActionButtonsProps = {
   actions: { redirectSQLLab: () => void; openPropertiesModal: () => void };
   canDownloadCSV: boolean;
   chartStatus: string;
-  latestQueryFormData: {};
+  latestQueryFormData: QueryFormData;
   queriesResponse: {};
   slice: { slice_name: string };
   addDangerToast: Function;
 };
 
+const VIZ_TYPES_PIVOT = ['pivot_table', 'pivot_table_v2'];
```

Comment:
nit: I wonder if we should call this `VIZ_TYPES_PIVOTABLE` to make it clear that it relates to whether or not the output can be pivoted or not, and not the chart type.

Reply:
Sounds good!